### DISCUSSION TITLE
v0.8.2

### DISCUSSION THREAD

**Discussion by mxcl** [teaxyz/teaBASE#45] [Announcements]
## What's Changed
* Improve pkgx binary detection with multiple locations by @stevenlei in https://github.com/teaxyz/teaBASE/pull/37

**Full Changelog**: https://github.com/teaxyz/teaBASE/compare/v0.8.1...v0.8.2

<hr /><em>This discussion was created from the release <a href='https://github.com/teaxyz/teaBASE/releases/tag/v0.8.2'>v0.8.2</a>.</em>

**Comment by Ayaan1982** [comment]
Nice 👍
